Makefile

```diff
@@ -1,7 +1,7 @@
 #   Makefile for T3
 
 install:
-	bash install_all.sh
+	bash devtools/install_all.sh
```

Comment:
same here, please squash (this line was added in the 1st commit, and is being fixed here)